<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R95C30S" failures="23" tests="23" name="test suit" time="0.379" errors="0" timestamp="2 Feb 2018 00:15:04 GMT">
  <testcase classname="test.TestHomePage" name="testGoToProfile" time="0.031">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.HomePage.getName(HomePage.java:19)
	at PageFactory.HomePage.nameClick(HomePage.java:23)
	at PageFactory.HomePage.goToProfile(HomePage.java:28)
	at test.TestHomePage.testGoToProfile(TestHomePage.java:35)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testGoToProfile -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.015">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.031">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.016">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.016">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.017">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.016">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.013">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.005">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.016">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.004">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.005">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.005">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.014">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.005">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.009">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="test.TestLoginPage" name="testLogin" time="0.006">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at PageFactory.LoginPage.setLog(LoginPage.java:21)
	at PageFactory.LoginPage.toLogin(LoginPage.java:34)
	at test.TestLoginPage.testLogin(TestLoginPage.java:25)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- test suit -->
